packages/core/src/accordion/accordion-panel.element.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { baseStyles, CdsBaseAccordionPanel } from '@clr/core/internal';
+import { ClarityIcons, angleIcon } from '@clr/core/icon';
+import { html } from 'lit-element';
+import { styles } from './accordion-panel.element.css.js';
+
+ClarityIcons.addIcons(angleIcon);
+
+/**
+ * Web component accordion
+ *
+ * ```typescript
+ * import '@clr/core/accordion/register.js';
+ * ```
+ *
+ * ```html
+ * <cds-accordion>
+ *   <cds-accordion-panel expanded='panel1Expanded' (expandedChange)='panel1Expanded = $event.detail'>
+ *    <cds-accordion-header>Item 1</cds-accordion-header>
+ *     <cds-accordion-content *ngIf='panel1Expanded'>Content 1</cds-accordion-content>
+ *   </cds-accordion-panel>
+ *   <cds-accordion-panel expanded='panel2Expanded' (expandedChange)='panel2Expanded = $event.detail'>
+ *     <cds-accordion-header>Item 2</cds-accordion-header>
+ *     <cds-accordion-content *ngIf='panel2Expanded'>Content 2</cds-accordion-content>
+ *   </cds-accordion-panel>
+ *
+ *   <cds-accordion-panel disabled>
+ *     <cds-accordion-header>Item 3</cds-accordion-header>
+ *     <cds-accordion-content>Content 3</cds-accordion-content>
+ *   </cds-accordion-panel>
+ * </cds-accordion>
+ * ```
+ *
+ * @element cds-accordion-panel
+ * @slot
+ * @slot cds-accordion-header
+ * @slot cds-accordion-content
+ * @event expandedChange - notify when the user has clicked the panel header
+ * @cssprop --border-color
+ * @cssprop --border-width
+ * @cssprop --color
+ * @cssprop --font-size
+ * @cssprop --font-weight
+ * @cssprop --header-background-color
+ * @cssprop --header-color
+ * @cssprop --header-font-size
+ * @cssprop --header-font-weight
+ * @cssprop --header-open-background-color
+ * @cssprop --transition-duration
+ */
+export class CdsAccordionPanel extends CdsBaseAccordionPanel {
+  connectedCallback() {
+    super.connectedCallback();
+    this.setAttribute('slot', 'accordion-panel');
+  }
+
+  private toggle() {
+    this.expanded = !this.expanded;
```

Comment:
Not sure if this was removed based on the comments above but looks like its still needs to be removed to prevent the stateful escape hatch.

Reply:
nvm I see it :)